index.bs

```diff
@@ -643,6 +643,19 @@ It has the following constraints:
 * Every element's [=report window/start=] is equal to the previous element's [=report window/end=], if it exists.
 * There is at least one element in the list.
 
+<h3 id="trigger-specs-header">Trigger specs</h3>
+
+A <dfn>trigger spec</dfn> is a [=struct=] with the following items:
+
+<dl dfn-for="trigger spec">
+: <dfn>event-level report windows</dfn>
+:: A [=report window list=].
+
```

Comment:
Should we also mention it could have a `start_time` field?

Reply:
The `report window list` structure already encompasses the start time: https://wicg.github.io/attribution-reporting-api/#report-window-header